ci-operator/config/openshift/installer/openshift-installer-main.yaml

```diff
@@ -1114,6 +1123,8 @@ tests:
     allow_best_effort_post_steps: true
     cluster_profile: azure4
     env:
+      AZURE_INSTALL_USE_MINIMAL_PERMISSIONS: "yes"
```

Comment:
Need to update pre to add `azure-provision-service-principal-minimal-permission`, and update post accordingly.